packages/buckets/src/utils.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { Buffer } from 'buffer'
+import { ListPathItem, ListPathReply } from '@textile/buckets-grpc/buckets_pb'
+import { bucketsListPath, BucketsGrpcClient } from './api'
+/**
+ * utilBufToArray converts a buffer into <4mb chunks for use with grpc API
+ * @param chunk an input Buffer
+ */
+export function bufToArray(chunk: Buffer, size = 1024 * 1024 * 3) {
```

Comment:
Since Buffers _are_ Uint8Arrays, it might be more generally useful to have this as Uint8Array. There's nothing in the code that would preclude this. You could still _return_ an array of buffers if you wanted?

Reply:
updated